app/main.js

```diff
@@ -2,6 +2,7 @@
 const {app, BrowserWindow, Tray, Menu, ipcMain, shell, dialog, globalShortcut} = require('electron')
 const i18next = require('i18next')
 const Backend = require('i18next-node-fs-backend')
+const { autoUpdater } = require("electron-updater")
```

Comment:
we use [standardjs](https://standardjs.com/), so this should be `require('electron-updater')`. there are more issues, try running `npm run lint` to see them

Reply:
@hovancik  Yes we can combine the  two checks in  one.  But the `function checkVersion()` only notifies the user  of a new release and is internally using `VersionChecker` for checking updates. This can be easily handled through electrons `autoUpdater.checkForUpdatesAndNotify()` .  In addition to this what we can do is instead of redirecting it to the github release page (as in current system), we can redirect it to a prompt to update the newer version.  
[Prompt](https://lh3.googleusercontent.com/-WgkKa0lCy6M/Wppv13sWaqI/AAAAAAAAa2o/-l6ssKBeLQQZbzLhZrCusMbSZhTyQ5qZwCL0BGAYYCw/h1080/2018-03-03.png)

Reply:
Yes, that's where I was pointing. As you have it right now, `checkForAutoUpdate()` becomes available in menu only when old version checker finds new version. 

The picture looks good ;]

Reply:
:+1:  Will fix this